scipy/spatial/distance.py

```diff
@@ -144,6 +146,88 @@ def _validate_vector(u, dtype=None):
         raise ValueError("Input vector should be 1-D.")
     return u
 
+def directed_hausdorff(u, v, seed=0):
+    """
+    Computes the directed Hausdorff distance between two N-D arrays.
+
+    Distances between pairs are calculated using a Euclidean metric.
+    
+    Parameters
+    ----------
+    u : (M,N) ndarray
+        Input array.
+    v : (O,N) ndarray
+        Input array.
+    seed : int or None
+        Local `RandomState` seed. Deafult is 0, a random shuffling of
+        u and v that guarantees reproducibility.
+
+    Returns
+    -------
+    out : tuple
```

Comment:
Seems better to document this as 3 separate returns. The tuple really isn't relevant.